packages/forma-36-website/src/pages/guidelines/copy/grammar/sentence-structure/index.mdx

```diff
@@ -0,0 +1,28 @@
+# Sentence structure
+
+## Focus on the objective of each sentence
+
+To make sentences scannable and easily understandable, start the sentence with the objective when possible.
+
+![Begin with the objective](/begin-with-objective.jpg)
+
+## Practice progressive disclosure
+
+To keep copy concise and actionable, eliminating extra details. Reveal details about a feature as the user interacts and requires more information.
```

Comment:
> To keep copy concise and actionable, eliminating extra details.`

I think this should be `To keep copy concise and actionable, eliminate extra details.`